dockerfiles/executor_image/Dockerfile

```diff
@@ -40,6 +40,11 @@ RUN DOCKER_VERSION="5:27.2.0-1~debian.12~bookworm" && \
     # keep these in sync with the binaries we are manually
     # adding in enterprise/server/cmd/executor/BUILD.
     #
+    # containerd
+    /usr/bin/containerd \
+    /usr/bin/containerd-shim \
+    /usr/bin/containerd-shim-runc-v1 \
+    /usr/bin/ctr \
```

Comment:
sanity check: have you tried running the executor via the docker image target and making sure it can still run a basic docker action?